Foxtrot/Foxtrot/Extraction/ExtractorVisitor.cs

```diff
@@ -1453,6 +1453,33 @@ public TypeNode ClosureClass
                 }
             }
 
+            public override void VisitReturn(Return Return)
```

Comment:
This is a name taken from base method. returnStatement seems to be best choice. Will chnage